packages/components/src/navigation/index.js

```diff
@@ -1,74 +1,53 @@
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
-import { Icon, arrowLeft } from '@wordpress/icons';
+import { useEffect, useState } from '@wordpress/element';
 
-/**
- * Internal dependencies
- */
-import Button from '../button';
-import Text from '../text';
-import Item from './item';
+const Navigation = ( { activeId, children, items: rawItems, rootTitle } ) => {
+	const [ activeLevel, setActiveLevel ] = useState( 'root' );
```

Comment:
The initial state of `activeLevel` is going to be the `activeId`.

Reply:
`activeId` represents the item ID as opposed to the level ID.  I renamed this `activeItemId` for the time being but we can rename later if that's still confusing.